source/_news/2025-08-11-roadmap.md

```diff
@@ -0,0 +1,44 @@
+---
+title: IIIF Specification roadmap
+author: Glen Robson
+date: 2025-08-11
+tags: [news, announcements]
+layout: post
+excerpt: >
+  Update on the plans for the IIIF version 4 specifications 
+---
+
+<img class="news-img-right" src="{{ "assets/images/news/2024-washington3d.png" | absolute_url}}" alt="Image of 3d meeting in Washington"/>
+The IIIF Editors have been revising the Presentation specification to incorporate support for 3D resources and have released an Alpha version of v4, now available here: [https://preview.iiif.io/api/prezi-4/presentation/4.0/](https://preview.iiif.io/api/prezi-4/presentation/4.0/). The IIIF editors are currently seeking feedback from the wider community ahead of the next in-person Editors meeting, scheduled for early December. The scope of v4, especially its support for 3D, was shaped during in-person meetings in Washington, D.C.; London; and Basel (with generous funding from the Kress Foundation for the D.C. meeting), as well as during ongoing online meetings of the Editors and the 3D TSG.
+
+
+The main focus over the next year will be on moving the v4 Alpha specification to a published release. This will involve the community implementing the draft specification and providing feedback. The 3D TSG community is very active in implementing the specifications as they are developed and it is likely we will see a number of implementations while changes are discussed. This is a marked difference to previous specification releases where implementation was done by limited implementations and so feedback was more difficult to collect.
+
+The [3D manifesto](https://github.com/IIIF-Commons/manifesto-3d) and blender work has also been a big benefit to encouraging early adoption and a number of viewers are using the manifesto library to start working with IIIF. The [blender IIIF plugin](https://github.com/IIIF-Commons/iiif_3d_blender_plugin) is also turning into a basic manifest editor and so will encourage experimentation and adoption.   
+
+In addition to the 3D TSG, the AV Motivations TSG are focused on methods for distinguishing between different types of AV annotations, such as subtitles, captions and transcriptions, to ensure that viewers can make use of these resources in a meaningful way that supports accessibility. The IIIF Editors have proposed a new property, “provides”, to support accessibility needs in IIIF v4. The group is currently developing cookbook recipes to demonstrate the intended use of “provides”. The recipes are almost ready but will have to wait for version 4 to be approved before going to the TRC. The group will then pause until version 4 is approved. 
```

Comment:
Do we want to mention the possible W3C accessibility sources of (some of the) values for the provides property, or do we deliberately NOT want to mention it? The Model doc does not (https://preview.iiif.io/api/prezi-4/presentation/4.0/model/#provides)
I think not